Problem1.cpp

```diff
@@ -0,0 +1,48 @@
+#include "pch.h"
+#include <gtest/gtest.h>
+
+using namespace std;
+
+int sumEvenNumbers(int[], int);
+
+int vector1[5] = { 2,13,15,6,2 };
+int vector2[10] = { 2,4,17,19,22,45,17,2,99,77 };
+int vector3[5] = { 3,5,7,9,11};
+int vector4[5] = { 2,4,6,8,10};
+int vector5[5];
+
+TEST(Sum_Even_Number_Tests, Test1) {
+	EXPECT_EQ(sumEvenNumbers(vector1, 5), 10);
+}
+
+TEST(Sum_Even_Number_Tests, Test2) {
+	EXPECT_EQ(sumEvenNumbers(vector2, 10), 30);
+}
+
+TEST(Sum_Even_Number_Tests, Test3) {
+	EXPECT_EQ(sumEvenNumbers(vector3, 5), 0);
+}
+
+TEST(Sum_Even_Number_Tests, Test4) {
+	EXPECT_EQ(sumEvenNumbers(vector4, 5), 30);
+}
+
+TEST(Sum_Even_Number_Tests, Test5) {
+	EXPECT_EQ(sumEvenNumbers(vector5, 5), 0);
+}
+
+TEST(Sum_Even_Number_Tests, Test6) {
```

Comment:
Where appropriate, can you change the test name into something that better defines what your test does?

Reply:
Yes, I can change the tests name